components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -26,6 +28,7 @@ export default class AdvancedSettingsDisplay extends React.Component {
             formatting: 'advancedCtrlSend',
             join_leave: 'formatting',
             advancedPreviewFeatures: 'join_leave',
+            deactivateAccount: 'deactivate_account',
```

Comment:
I think this should be:
```
deactivateAccount: 'advancedPreviewFeatures',
```
It is used to focus on `Edit` after previous sections has been saved/updated.

Reply:
done